[DOCS-1687] Release notes for Server v0.72.0 (#1487)
wandb/docs: content/en/ref/release-notes/releases/0.71.md

- Fixed wrapping of long names for Sweeps and Reports.
- Improved the copy and paste behavior for console logs. Line numbers are no longer copied, and timestamps now appear inline instead of on a new line.

## Patches
### 0.71.1
**August 6, 2025**
- Reverted the line plot behavior change announced in 0.71.0. Line plots with only 1 point for every line will convert to bar charts even if settings are changed.
- Fixed a bug that could cause bounding boxes to render outside the media in a media panel instead of overlaying the media.

> - Weave traces submitted through Google Agent Development Kit or OpenTelemetry now translate the ADK format into a single JSON blob that includes both the request and response, and the chat view now displays both.

Author: ```suggestion
- Fixed a Weave bug where Google Agent Development Kit traces submitted through OpenTelemetry displayed as JSON blobs instead of chat contents.
```

> - Fixed a bug where the **Attributes** column was missing from Weave submitted through OpenTelemetry.

Author: ```suggestion
- Fixed a bug where the **Attributes** column was missing from Weave traces submitted through OpenTelemetry.
```

- Fixed wrapping of long names for Sweeps and Reports.
- Improved the copy and paste behavior for console logs. Line numbers are no longer copied, and timestamps now appear inline instead of on a new line.

## Patches
### 0.71.1
**August 6, 2025**
- Reverted the line plot behavior change announced in 0.71.0. Line plots with only 1 point for every line will convert to bar charts even if settings are changed.

> - Fixed a bug that could cause bounding boxes to render outside the media in a media panel instead of overlaying the media.

Author: ```suggestion
- Fixed a bug that could cause bounding boxes to render outside the media in a media panel instead of overlaying the media.
- Weave traces submitted through Google Agent Development Kit or OpenTelemetry now translate the ADK format into a single JSON blob that includes both the request and response, and the chat view now displays both.
- Fixed a bug where the **Attributes** column was missing from Weave submitted through OpenTelemetry.
```
